.travis.yml

```diff
@@ -1,25 +1,31 @@
 language: php
+sudo: false
```

Comment:
I'm not sure if where Humbug falls on sudo defaults, though I almost assume it defaults to false for the new humbug/humbug repo on Travis. Made in explicit just in case.